src/app/collections/PresentationNode.scss

```diff
@@ -5,7 +5,7 @@
     margin-right: 4px;
   }
 
-  > .presentation-node > .presentation-node {
+  > .presentation-node {
```

Comment:
I think the reason I did this is because Collections look weird. Can you take a look there?

Reply:
this is because collections header itself is an exceptionally styled, uncollapsible top level presentation node, and i am fixing this along with some other presentation node quirks

Reply:
this is so poorly described it is incorrect, but suffice it's being included